src/components/task-plan/homework/exercises.cjsx

```diff
@@ -182,7 +183,10 @@ ExerciseTable = React.createClass
 
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
-    content = ExerciseStore.getContent(exerciseId)
+    content = $("<span>#{ExerciseStore.getContent(exerciseId)}</span>")
+
+    _.each(content.find('img'), (img) -> $(img).before("<span>[image] #{img.alt}</span>"))
```

Comment:
I'm not sure if it is clear to the user that the text is actually alt text and not part of the question.

thoughts @openstax/ux ?